jwst/extract_1d/extract_1d_step.py

```diff
@@ -340,13 +339,41 @@ def _extract_soss(self, model):
 
         return result
 
+    def _check_mrs_type(self,model):
+        """ Check if the MIRI MRS data is for a single band """
+        # check channel is only 1 value and band is only 1 value
+        validch = ['1', '2', '3', '4']
+        validband = ['short', 'medium', 'long']
+        band_check = False
+        ch_check = False
+        b_check = False
+        if model.meta.instrument.channel  in validch:
+            ch_check = True
+        if model.meta.instrument.band.lower()  in validband:
```

Comment:
Add a safety check here for `None` in the band value:

```suggestion
        if str(model.meta.instrument.band).lower() in validband:
```
I think this is causing the current unit test failures.